src/App.tsx

```diff
@@ -11,13 +11,19 @@ import { ThemeProvider } from "@fluentui/react";
 import { FluentProvider, webLightTheme } from "@fluentui/react-components";
 import { UserProvider } from "providers/UserProvider";
 import { lazy, Suspense } from "react";
+import Home from "Home";
```

Comment:
Suggest removing unused file App.css and assets folder which contains usused react.svg